packages/fxa-auth-server/test/local/routes/devices-and-sessions.js

```diff
@@ -848,6 +853,26 @@ describe('/account/devices/invoke_command', () => {
         'notifyMessageReceived was not called');
     });
   });
+
+  it('supports feature-flag for oauth devices', async () => {
+    const mockPushbox = mocks.mockPushbox();
+    const route = getRoute(makeRoutes({
+      config: { oauthDeviceCommandsEnabled: false },
+      customs: mockCustoms,
+      log: mockLog,
+      pushbox: mockPushbox,
+    }), '/account/device/commands');
```

Comment:
Should this be `/account/devices/invoke_command`?

Reply:
nice catch!